anasymod/fpga_boards/boards.py

```diff
@@ -18,9 +18,11 @@ class PYNQ_Z1(FPGA_Board):
     board_part = 'www.digilentinc.com:pynq-z1:part0:1.0'
     full_part_name = 'xc7z020clg400-1'
     short_part_name = 'xc7z020'
+    bram = 4.9e6
+    dbg_hub_clk_freq = 100e6
     fpga_sim_ctrl = [FPGASimCtrl.UART_ZYNQ, FPGASimCtrl.VIVADO_VIO]
-    uart_vid = 1027
-    uart_pid = 24592
+    uart_zynq_vid = [1027]

```

Comment:
I think this name change likely has to be updated for the other boards as well; would you mind doing that?  Thanks.